package.json

```diff
@@ -552,6 +563,17 @@
                     "generic"
                   ]
                 },
+                "autoSelect": {
+                  "type": "boolean",
+                  "markdownDescription": "If true, this sequence will be automatically selected, suppressing the Project Type menu when connecting to a REPL or Jacking in. Use together with `projectRootPath` to also suppress the Project Root menu. Add usage of `menuSelections` to go for a prompt-less REPL connection. If you have more than one sequence with `autoSelect` set to true, the first one will be used."
+                },
+                "projectRootPath": {
+                  "type": "array",
+                  "markdownDescription": "An array of path segments leading to the root of the project for which this connect sequence should go with. Use together with `autoSelect` to suppress the Project Root menu. The path can be absolute or relative to the workspace root. If there are several Workspace Folders, the workspace root is the path of the first folder. So relative paths will only work for this first folder. If you can't be bothered splitting up the path in segments, put the whole path in the first segment, though please note that if you use Windows path separators, these will not work for users with Linux or macOS.",
+                  "items": {
+                    "type": "string"
+                  }
+                },
                 "nReplPortFile": {
                   "type": "array",
                   "description": "An array of path segments with the project root-relative path to the nREPL port file for this connect sequence. E.g. For shadow-cljs this would be [\".shadow-cljs\", \"nrepl.port\"]",
```

Comment:
```suggestion
                  "description": "An array of path segments for the relative path from the project root to the nREPL port file for this connect sequence. E.g. For shadow-cljs this would be [\".shadow-cljs\", \"nrepl.port\"]",
```